src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+const defaultTodo = {
+  id: 0,
+  title: 'no title',
+  completed: false,
+  userId: 0,
+};
```

Comment:
you already have this object on App component. But don't need to import it, just use null like default value